js/interface/map-panel.js

```diff
@@ -22,7 +22,50 @@ Fliplet.InteractiveMap.component('map-panel', {
       default: true
     }
   },
+  data() {
+    return {
+      updateDebounced: _.debounce(this.updateDataSource, 1000),
+      appId: _.keys(__widgetData)[0],
```

Comment:
This definitely won't work. The first key in `__widgetData` is not the `appId` but the first widget instance ID in the page.

Please explain.

Reply:
@squallstar My bad. Forgot to use fliplet methods. Fixed now.